src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -10,5 +75,18 @@ NuGet.Commands.Restore.IProject.TargetFrameworks.get -> System.Collections.Gener
 NuGet.Commands.Restore.ITargetFramework
 NuGet.Commands.Restore.ITargetFramework.GetItems(string! itemType) -> System.Collections.Generic.IReadOnlyList<NuGet.Commands.Restore.IItem!>!
 NuGet.Commands.Restore.ITargetFramework.GetProperty(string! propertyName) -> string!
+NuGet.Commands.Restore.ProjectBuildProperties
+NuGet.Commands.Restore.ProjectItems
 NuGet.Commands.Restore.Utility.PackageSpecFactory
+NuGet.ProjectManagement.ProjectItemProperties
```

Comment:
As long as the two VS project system implementations use these types, they still need to be public APIs.

If we're willing to break public APIs, then in the future when we delete the old PackageSpec building methods and only have PackageSpecFactory left, then we could consider making these internal.  But at the same time, the tech debt of public const strings that can't be changed for breaking changes reasons is very low.